argusclient/client.py

```diff
@@ -300,8 +302,8 @@ def update_users(self, namespaceid, *users):
 
 
 class BaseUpdatableModelServiceClient(BaseModelServiceClient):
-    def __init__(self, objType, argus, get_all_path, id_path):
-        super(BaseUpdatableModelServiceClient, self).__init__(argus, get_all_path)
+    def __init__(self, objType, argus, get_all_path, id_path, get_all_path_params=None):
```

Comment:
How about switching `get_all_path` and `id_path` around so that the two get_all parameters are together? This would actually be a non-backward compatible change, but I doubt if there are any references outside our library. We can bump up the major version to indicate that this is not backwards compatible.

Reply:
Sure, ill make the change